File: p006/p006.js
function sumSquareDifference(n) {
  // Good luck!
  let squareOfSum = ((n * (n + 1)) / 2) ** 2,
    sumOfSquares = (n * (n + 1) * (2 * n + 1)) / 6;
  return squareOfSum - sumOfSquares;
}

sumSquareDifference(100);

File: p006/p006.py
def sumSquareDifference(n):
    squareOfSum = (n * (n + 1) / 2) ** 2
    sumOfSquares = (n * (n + 1) * (2 * n + 1)) / 6
    return int(squareOfSum - sumOfSquares)

if __name__ == "__main__":
    print(sumSquareDifference(100))